### PR TITLE
Add config param for url of loading gtm.js

### PR DESCRIPTION
**Pull request by NorthBei**
Because I have mutliple types of environment in GTM, for example, Live(production), Staging, Dev, etc.
To use different environment, I should load gtm.js with different url query string respectively in my website.
So, I add a new config parameter named 'queryParams' ,let eveyone can use this parameter to append some query string to the url which use for load gtm.js.
And I update the README.md with new tutorial about this feature and bulid new vue-gtm.min.js file.
Hope that you can merge and update the vue-gtm on npm ASAP, thank you.

### CODE REVIEW

File: src/utils.js [hunk 1]
```diff
@@ -31,8 +31,13 @@ export const loadScript = function (id) {
     return
   }
 
+  let querys = []
+  for (const [key, value] of Object.entries(queryParams)) {
+    querys.push(key + '=' + value)
+  }
+
```
Comment (ksorokin) [inline src/utils.js:37]: Thanks for your PR, I also looking for the solution to pass extra variables into config. 

There is could be more graceful solution 
```
const searchParams = new URLSearchParams(queryParams)
searchParams.append('id', id)

script.src   = `https://www.googletagmanager.com/gtm.js?${searchParams.toString()}`
```

**Comment by ksorokin** [comment]
@mib200 please, merge PR

**Comment by ALiangLiang** [comment]
LGTM. @mib200 pls merge this PR.

File: src/utils.js (hunk 1 above)
Comment (NorthBei) [reply src/utils.js:37]: Oh, I think it's more graceful implementation let the code more readable.
I already use your graceful code to replace my code in next commit.
Thanks for your suggestion!

**Comment by assembledadam** [comment]
+1

**Comment by mib200** [comment]
committed to master and closed.

**Comment by NorthBei** [comment]
> committed to master and closed.

Hello, I found you have been merge code manually in this commit (0eb8ac8d568399dcbf6a37308e52c7a829f4d156)
You already add the "queryParams" arugment in config.js and index.js.
And in index.js, you pass the merged config's "queryParams" to "loadScript" function where is located in utils.js
but in the utils.js, the "loadScript" function never used the "queryParams" parameter in the, so we can't use config's queryParams to append the querystring to url which used to load gtm.js...

Maybe you can merge this code in the "loadScript" function to use "queryParams" object !

```
const searchParams = new URLSearchParams(queryParams)
searchParams.append('id', id)

script.src   = `https://www.googletagmanager.com/gtm.js?${searchParams.toString()}`
```

**Comment by Hsueh-Jen** [comment]
Hasn't this issue been fixed ? how sad. QQ

**Pull request closed without merging**